web-components/src/components/form/CreditRetireForm.tsx

```diff
@@ -218,7 +209,7 @@ export const initialValues = {
 };
 
 const CreditRetireForm: React.FC<FormProps> = ({
-  holder,
+  // holder,
```

Comment:
this prop is currently unused so I commented it out, but looks like it might be wired up later?

Reply:
no I think it's actually unused, should be removed but I can take care of that in an upcoming PR since I'm gonna work on those forms anyway as part of https://app.zenhub.com/workspaces/regen-registry-5f8998bec8958d000f4609e2/issues/regen-network/regen-registry/884